analyses/doublet-detection/scripts/01a_run-scdblfinder.R

```diff
@@ -7,18 +7,20 @@
 project_root <- rprojroot::find_root(rprojroot::is_renv_project)
 renv::load(project_root)
```

Comment:
You need to remove `renv::load()` commands from your scripts to prevent loading when not wanted.

Reply:
omg 🤦🏻‍♀️